server/sonar-server/src/main/java/org/sonar/server/qualitygate/ws/ProjectStatusAction.java

```diff
@@ -57,17 +69,19 @@ public String apply(@Nonnull ProjectStatusWsResponse.Status input) {
     }));
 
   private final DbClient dbClient;
+  private final ComponentFinder componentFinder;
   private final UserSession userSession;
 
-  public ProjectStatusAction(DbClient dbClient, UserSession userSession) {
+  public ProjectStatusAction(DbClient dbClient, ComponentFinder componentFinder, UserSession userSession) {
     this.dbClient = dbClient;
+    this.componentFinder = componentFinder;
     this.userSession = userSession;
   }
 
   @Override
   public void define(WebService.NewController controller) {
     WebService.NewAction action = controller.createAction("project_status")
-      .setDescription(String.format("Quality gate status for a given Compute Engine task. <br />" +
+      .setDescription(String.format("Quality gate status for a given project or Compute Engine task. <br />" +
```

Comment:
The description is not clear. It does not highlight which parameters must be set.